src/drivers/scr_server/scr_server.cpp

```diff
@@ -247,8 +248,10 @@ newrace(int index, tCarElt* car, tSituation *s)
     // Wait for connections from clients.
     listen(listenSocket[index], 5);
 
-    std::cout << "Waiting for request on port " << UDP_LISTEN_PORT+index << "\n";
-
+    if(const char* env_p = std::getenv("TORCS_PORT"))
+	    std::cout << "Waiting for request on port " << std::atoi(env_p)+index << "\n";
+    else
+    	std::cout << "Waiting for request on port " << UDP_LISTEN_PORT+index << "\n";
```

Comment:
Is this the only change you have made, if so, then how is the functionality working ? This is a change in the print statement and no where else could I find the usage of the variable env_p.